languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/requests/JavaHttpRequestRenderer.kt

```diff
@@ -191,15 +193,20 @@ class JavaHttpRequestRenderer @Inject constructor(override val vrapTypeProvider:
             |   httpRequestPath += "?" + String.join("&", params);
             |}
         """.trimMargin().escapeAll()
-        
+        val bodySetter: String = if(bodyName != null){
+            if(this.bodies[0].type.isFile())
+                "try{httpRequest.setBody(Files.readAllBytes(file.toPath()));}catch(Exception e){e.printStackTrace();}"
```

Comment:
I see 👍 But wouldn't it be better to use a logger here?

Reply:
Nice catch, will see this 👍